generated/niscope/niscope/_grpc_stub_interpreter.py

```diff
@@ -250,9 +250,11 @@ def fetch_into_numpy(self, channel_list, timeout, num_samples):  # noqa: N802
         raise NotImplementedError('numpy-specific methods are not supported over gRPC')
 
     def fetch_array_measurement(self, channel_list, timeout, array_meas_function, measurement_waveform_size):  # noqa: N802
+        if measurement_waveform_size is not None:
+            raise ValueError('The argument "meas_wfm_size" should be None when using gRPC.')
```

Comment:
should -> must

Reply:
Switched to must in the error message and corresponding system test assertion.